### ISSUE TITLE
Can I use API to generate Music given lyrics as condition?

### ISSUE THREAD

**Issue by puar-playground** [gcui-art/suno-api#157]
**Is your feature request related to a problem? Please describe.**
Can I use API to generate Music given lyrics as condition?

**Describe the solution you'd like**
Give a demo code of how to input lyrics for music generation

**Comment by glitchbunny0** [comment]
Yes — `POST /api/custom_generate` accepts lyrics as the `prompt` field (Custom Mode). You can also pair it with `tags` for style and `title` for the song name. Example in the [examples/](https://github.com/glitchbunny0/suno-api/tree/main/examples) directory of the fork.

Additionally, the fork has `POST /api/cowrite_lyrics` (AI-assisted lyric writing from a prompt) and `POST /api/generate_lyrics` (full lyric generation) if you need help crafting the lyrics first.
